sklearn/metrics/tests/test_ranking.py

```diff
@@ -354,7 +354,7 @@ def test_roc_curve_drop_intermediate():
     # Test that drop_intermediate drops the correct thresholds
     y_true = [0, 0, 0, 0, 1, 1]
     y_score = [0., 0.2, 0.5, 0.6, 0.7, 1.0]
-    tpr, fpr, thresholds = roc_curve(y_true, y_score)
+    tpr, fpr, thresholds = roc_curve(y_true, y_score, drop_intermediate=True)
```

Comment:
Just a note: whatever the default value we end up for `drop_intermediate` it's a good idea to force it to `True` in the tests to make it explicit that we test the behavior of this specific parameter.